redis/cache-redis.conf

```diff
@@ -0,0 +1,13 @@
+# allow access from all instances
```

Comment:
Looks like the redis conf changes snuck in by accident!

Reply:
!!! NO!!!! bad redis conf!